telemetry/specs/INT.mdk

```diff
@@ -711,6 +711,26 @@ INT shim header for VXLAN GPE encapsulation:
 * Type: This field indicates the type of INT Header following the shim header.
 Two Type values are used: one for the hop-by-hop header type and the other for
 the destination header type (See Section [#sec-int-header-types]).
+* G: Indicates whether the original packet (before insertion of INT headers
+and metadata) used a VXLAN or VXLAN GPE encapsulation.
+  - 0: Original packet used VXLAN encapsulation.
+  - 1: Original packet used VXLAN GPE encapsulation.
```

Comment:
Wondering whether we should flip this and use 0 to indicate the original packet was VXLAN GPE, i.e. no conversion is required and 1 to indicate that original packet was VXLAN, i.e. yes conversion is required at INT sink. Thus INT sink does VXLAN GPE to VXLAN conversion if this bit is set to 1.

Reply:
I don't have a very strong preference but using 1 to indicate the need for conversion seems a bit more indicative and useful for debugging. Again, no strong opinion.

Reply:
Done